### PR TITLE
Upgrade to NET 6.0 (v2.2.0)

### PR DESCRIPTION
**Pull request by fdrobidoux**

### CODE REVIEW

File: Monofoxe/Monofoxe.Pipeline/Monofoxe.Pipeline.csproj [hunk 1]
```diff
@@ -1,21 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<TargetFramework>net6.0</TargetFramework>
 		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
 		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
 		<OutputPath>..\bin\$(Configuration)\</OutputPath>
 		<NoWarn>NU5100</NoWarn>
+		<Version>2.2.0</Version>
```
Comment (Martenfur) [inline Monofoxe/Monofoxe.Pipeline/Monofoxe.Pipeline.csproj]: Remove that, versions are set elsewhere.

Comment (Martenfur) [reply Monofoxe/Monofoxe.Pipeline/Monofoxe.Pipeline.csproj]: Everywhere else too.

File: Samples/Monofoxe.Samples.WindowsDX/Monofoxe.Samples.WindowsDX.csproj [hunk 2]
```diff
@@ -14,8 +14,8 @@
 		<TrimmerRootAssembly Include="Microsoft.Xna.Framework.Content.ContentTypeReader" Visible="false" />
 	</ItemGroup>
 	<ItemGroup>
-		<PackageReference Include="MonoGame.Framework.WindowsDX" Version="3.8.0.1641" />
-		<PackageReference Include="MonoGame.Content.Builder.Task" Version="3.8.0.1641" />
+		<PackageReference Include="MonoGame.Framework.WindowsDX" Version="3.8.1.303" />
+		<PackageReference Include="MonoGame.Content.Builder.Task" Version="3.8.1.303" />
 		<PackageReference Include="Nopipeline.Task" Version="2.1.5" />
```
Comment (Martenfur) [inline Samples/Monofoxe.Samples.WindowsDX/Monofoxe.Samples.WindowsDX.csproj]: Update Nopipeline to 2.2.0.

File: Monofoxe/Monofoxe.Engine.DesktopGL/Monofoxe - Backup.Engine.DesktopGL.csproj [hunk 3]
```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (Martenfur) [inline Monofoxe/Monofoxe.Engine.DesktopGL/Monofoxe - Backup.Engine.DesktopGL.csproj:1]: What does this do here?

**Review by Martenfur — CHANGES_REQUESTED** [review]

**Comment by Martenfur** [comment]
Also add an entry to CHANGELOG.md.

File: Monofoxe/Monofoxe.Pipeline/Monofoxe.Pipeline.csproj (hunk 1 above)
Comment (fdrobidoux) [reply Monofoxe/Monofoxe.Pipeline/Monofoxe.Pipeline.csproj]: Oh shit forgot you do that with `$(MonofoxeVersion)`. Sorry.

File: Monofoxe/Monofoxe.Engine.DesktopGL/Monofoxe - Backup.Engine.DesktopGL.csproj (hunk 3 above)
Comment (fdrobidoux) [reply Monofoxe/Monofoxe.Engine.DesktopGL/Monofoxe - Backup.Engine.DesktopGL.csproj:1]: DesktopGL is not necessarily for Windows.

**Comment by fdrobidoux** [comment]
> Also add an entry to CHANGELOG.md.

Sure.

File: Monofoxe/Monofoxe.Engine.DesktopGL/Monofoxe - Backup.Engine.DesktopGL.csproj (hunk 3 above)
Comment (fdrobidoux) [reply Monofoxe/Monofoxe.Engine.DesktopGL/Monofoxe - Backup.Engine.DesktopGL.csproj:1]: But sure, I'll add it back. I'll just remove it for myself.

**Pull request merged**